moveit_ros/planning/trajectory_execution_manager/test/test_subscriber.cpp

```diff
@@ -0,0 +1,113 @@
+#include <iostream>
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/string.hpp"
+
+class MinimalSubscriber : public rclcpp::Node
+{
+public:
+  MinimalSubscriber() : Node("trajectory_execution_manager_minimal_subscriber")
+  {
+    rmw_qos_profile_t qos = rmw_qos_profile_default;
+    qos.depth = 1;
+    qos.durability = RMW_QOS_POLICY_DURABILITY_TRANSIENT_LOCAL;
+
+    subscription_ = this->create_subscription<std_msgs::msg::String>(
+        "robot_description",
+        [this](const std_msgs::msg::String::ConstSharedPtr msg) {
+          RCLCPP_INFO(this->get_logger(), "I heard: '%s'", msg->data.c_str());
+        },
+        qos);
+  }
+
+private:
+  rclcpp::Subscription<std_msgs::msg::String>::SharedPtr subscription_;
+};
+
+int main(int argc, char* argv[])
+{
+  // Force flush of the stdout buffer.
+  setvbuf(stdout, NULL, _IONBF, BUFSIZ);
+
+  rclcpp::init(argc, argv);
+  rclcpp::spin(std::make_shared<MinimalSubscriber>());
+  rclcpp::shutdown();
+  return 0;
+}
+//
+// #include "rclcpp/rclcpp.hpp"
```

Comment:
Do you think it's worth enabling and fixing some of these tests? Otherwise, they should be removed

Reply:
It's not clear which robot model is used in the original tests. But I think some of these added tests are not valuable for this package. Removed these tests and resumed the original tests. I think maybe it's better to open a new PR to extend the original tests when moveit_ros_planning is ready.